src/packages/cli/src/__tests__/studio.test.ts

```diff
@@ -21,7 +21,7 @@ let studio: Studio
 const describeIf = (condition: boolean) =>
   condition ? describe : describe.skip
 
-describeIf(process.env.PRISMA_FORCE_NAPI !== 'true')('studio', () => {
+describe('studio', () => {
```

Comment:
There is a PR for this here https://github.com/prisma/prisma/pull/7680

Reply:
Nice, then merge that. It seems to generally pass here as well, so should not be a problem.